pilot/pkg/config/memory/config.go

```diff
@@ -233,3 +233,7 @@ func (cr *store) Update(cfg config.Config) (string, error) {
 	ns.Store(cfg.Name, cfg)
 	return rev, nil
 }
+
+func (cr *store) UpdateStatus(cfg config.Config) (string, error) {
+	return "", nil
```

Comment:
Is there a reason in memory resources can't have status?

Reply:
This would probably be useful in tests.